segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessor.java

```diff
@@ -57,15 +56,12 @@
     private final AtomicReference<EventStreamWriter<AutoScaleEvent>> writer;
     private final EventWriterConfig writerConfig;
     private final AutoScalerConfig configuration;
-    private final Executor executor;
```

Comment:
Hmm, why did we have this redundant executor?

Reply:
Good question....